rpcs3/Emu/System.h

```diff
@@ -221,7 +222,7 @@ class Emulator final
 
 	game_boot_result Load(const std::string& title_id = "", bool add_only = false, bool force_global_config = false, bool is_disc_patch = false);
 	void Run(bool start_playtime);
-	bool Pause();
+	bool Pause(bool no_resume = false);
```

Comment:
Why not just call it on_fatal_error

Reply:
I renamed it to embalm_state because I think it best describes what it does.

Reply:
this is even worse lmao

Reply:
The latest commit shows a different usage of it, not just for fatal errors, but purely for preserving emulation state forever until killed.

Reply:
I think the word: frozen probably makes more sense.